clients/spark/core/src/test/resources/parser-test/sst_files_generator.go

```diff
@@ -100,13 +100,12 @@ func newGenerateFuzz() func() (string, error) {
 
 func newGenerateFuzzWithSharedPrefixes() func() (string, error) {
 	f := fuzz.NewWithSeed(FuzzerSeed)
-	src := rand.NewSource(041021)
+	src := rand.NewSource(20211004)
 	r := rand.New(src)
-	prefixBaseLen := 1 + r.Intn(100)
-	sharedPrefixBase, err := nanoid.New(prefixBaseLen)
-	if err != nil {
-		panic(err)
-	}
+	prefixBaseLen := 1 + r.Intn(10)
+	sharedPrefixBaseBytes := make([]byte, prefixBaseLen)
+	f.Fuzz(&sharedPrefixBaseBytes)
```

Comment:
I think that https://play.golang.org/p/ahKo2Y4YjNW works but this actually does not.

This actually does _not_ work: slices are not the same as arrays, and in particular the fuzz library takes only OUT parameters, so _it doesn't look at their currently allocated capacity or length_.  E.g. https://play.golang.org/p/HNZiBea8y_B does not manage to get the intended size.